corehq/apps/app_manager/tests/test_suite_remote_request.py

```diff
@@ -731,3 +731,19 @@ def test_allow_blank_value(self, *args):
         </partial>
         """
         self.assertXmlPartialEqual(expected, suite, "./remote-request[1]/session/query/prompt[@key='name']")
+
+    def test_exclude_from_search(self, *args):
+        self.module.search_config.properties[0].exclude = True
+        suite = self.app.create_suite()
+        expected = """
+        <partial>
+          <prompt key="name" exclude="true()">
```

Comment:
Is this supposed to be `true()` not `true`?  In this same file, other attributes are shown to be the latter:
https://github.com/dimagi/commcare-hq/blob/025928ebcb382e9636e6e62264be5a4e4e00606e/corehq/apps/app_manager/tests/test_suite_remote_request.py#L626-L626
https://github.com/dimagi/commcare-hq/blob/025928ebcb382e9636e6e62264be5a4e4e00606e/corehq/apps/app_manager/tests/test_suite_remote_request.py#L765-L765

I'm guessing that's also why the xml model is defined as a StringField - to be able to pass in `true()`.

Reply:
Initially it was defined as a Boolean in the model. I changed it in order for it to be processed as an xpath expression as per Simon's [comment in the spec](https://docs.google.com/document/d/1FeHaa1JoA6NCRJfPX8Nt3joOAlqzFEgpT0NKlZkgy1I/edit?disco=AAAAXIoO5Yw).

Reply:
I see, I'd missed that, thanks